modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -592,13 +591,15 @@ public void batchIncomplete() {
             }
         }
 
-        Throwable ex = assertThrowsWithCause(
-                () -> ses.executeBatchAsync(null, "INSERT INTO TEST VALUES (?, ?)", args).join(),
-                SqlBatchException.class
+        CompletionException ex = assertThrows(
```

Comment:
Should we change CompletionException to SqlExeption?

Reply:
This is not an option because `assertThrows` verifies the topmost exception only. `CompletionException` in this particular case is caused by `join()` within the closure rather then by internals of sql engine